scalafix-testkit/src/main/scala/scalafix/testkit/SyntacticRewriteSuite.scala

```diff
@@ -16,17 +15,14 @@ class SyntacticRewriteSuite(rewrite: Rewrite)
   def check(name: String, original: String, expected: String): Unit = {
     test(name) {
       import scala.meta._
-      val obtained =
-        Scalafix
-          .fix(Input.String(original), ScalafixConfig(rewrite = rewrite))
-          .get
+      val obtained = rewrite.apply(Input.String(original))
```

Comment:
Awesome, I'm glad you like it. Yeah, I'm open to adding overloads for strings and java.io.File.